app/controllers/concerns/oai_concern.rb

```diff
@@ -61,5 +89,38 @@ def oai_xmlns
         'xsi:schemaLocation' => 'http://www.openarchives.org/OAI/2.0/ http://www.openarchives.org/OAI/2.0/OAI-PMH.xsd'
       }
     end
+
+    # XML namespace values for OAI-PMH Dublin Core containers
+    # Used for the ListSets description, see:
+    # http://www.openarchives.org/OAI/openarchivesprotocol.html#ListSets
+    def oai_dc_xmlns
+      {
+        'xmlns:oai_dc' => 'http://www.openarchives.org/OAI/2.0/oai_dc/',
+        'xmlns:dc' => 'http://purl.org/dc/elements/1.1/',
+        'xmlns:xsi' => 'http://www.w3.org/2001/XMLSchema-instance',
+        'xsi:schemaLocation' => 'http://www.openarchives.org/OAI/2.0/oai_dc/ http://www.openarchives.org/OAI/2.0/oai_dc.xsd'
+      }
+    end
+
+    def list_sets_description(stream)
```

Comment:
I'm not sure if we have any other use for it, but I'm finding this a little hard to read.

I wonder if this would be easier to write as `Stream#oai_description`? Or, maybe just more compactly as something like:

```
def list_sets_description(stream)
  "#{stream.default? ? 'Current' : 'Former'} default stream for #{stream.organization.slug} #{list_sets_date_ranges.join(', ')"
end
```

Reply:
Moved this to `Stream` and tweaked our `ListSets` response slightly so that the dates a stream was default and its organization slug are a bit more machine-parseable (moved them to dublin core fields within the description). Also added tests for the dublin core fields. cc @corylown

Reply:
pic for reference:

![Screen Shot 2022-05-25 at 15 48 33](https://user-images.githubusercontent.com/4924494/170382026-a632141c-e8ed-43c7-a78a-c236b9165083.png)